R/build_lookup.R

```diff
@@ -62,6 +62,39 @@ parse_imgt_fasta <- function(infile) {
   imgt_list
 }
 
+#' Extract gene names from a reference CSV for a given species
+#'
+#' `parse_mixcr_csv()` generates mixcr gene name data for a given species.
+#' @param species A string specifying the species, one of "human", "mouse",
+#' or "rhesus".
+#'
+#' @return A character vector of gene names.
+#' @export
+#' @keywords internal
+#' @examples
+#' parse_mixcr_csv("human")
+parse_mixcr_csv <- function(species) {
+  ## Throw error if input is not one of "human", "mouse", or "rhesus"
+  if (!species %in% c("human", "mouse", "rhesus")) {
+    stop("Input must be one of \'human\', \'mouse\', or \'rhesus\'")
+  }
```

Comment:
I don't think we should have this check in case a user has a custom species reference. I would take this out
```suggestion
```